flang/docs/tutorials/addingIntrinsics.md

```diff
@@ -277,6 +277,13 @@ It defines that when presented with a call to an intrinsic named "trim",
 that is not elemental, lowering must lower its `string` argument in memory, and pass this lowered
 argument to a function called `genTrim` that implements the actual lowering of TRIM.
 
+Pass by value is the default argument passing scheme. If all arguments are passed by value, then you do not need to specify any arguments in this table. If at least one argument is not passed by value, then it is good practice to specify all the arguments. For example, consider the genScan entry:
+```c++
+{"scan", &I::genScan, {{ {"string", asAddr}, {"set", asAddr}, 
+                         {"back", asAddr}, {"kind", asValue} }}, 
+                         /*isElemental*/ true}
```

Comment:
LLVM style seems to be `/*isElemental=*/` (with the equals).

Reply:
OK. I copied this from "trim" example. I can fix that too.